libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.spec.ts

```diff
@@ -28,8 +28,9 @@ import { DhUsersTabComponent } from './dh-users-tab.component';
 import { searchDebounceTimeMs } from './dh-users-tab-search.component';
 import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
 import { MatSelectHarness } from '@angular/material/select/testing';
+import { UserStatus } from '@energinet-datahub/dh/shared/domain';
 
-describe('DhUsersTabComponent.name', () => {
+describe(DhUsersTabComponent.name, () => {
```

Comment:
When I did this, the test would not run locally. So ended up with 'DhUsersTabComponent'